components/03-sections/01-accordion/accordion.scss

```diff
@@ -40,7 +40,7 @@
 		font-size: 21px;
 		font-weight: 600;
 		margin-bottom: 0;
-		padding: 20px 31px;
+		padding: 20px 60px 20px 30px;
 		position: relative;
```

Comment:
let's also add a sane equivalent in the XS breakpoint (down at 116)
Can we add slightly less padding on mobile? =)
<img width="329" alt="Screenshot 2022-07-26 at 2 23 46 PM" src="https://user-images.githubusercontent.com/16828076/181094653-2f3aa769-99c3-47f2-8488-422492a1da14.png">